app/models/label/LabelTable.scala

```diff
@@ -464,6 +464,14 @@ object LabelTable {
         "LEFT JOIN ( SELECT NULL AS validation_result ) AS user_validation ON lb.label_id = NULL"
       }
 
+    // Query to get comments and make them into a list
+    val commentsJoin: String =
```

Comment:
You don't need to define this as a variable and then combine it with the big query string, you can just put this directly into the query! The only reason we are defining pieces outside of the big query are in situations where we want to change the query based on input.